core/src/test/scala/unit/kafka/admin/ConfigCommandTest.scala

```diff
@@ -290,14 +340,9 @@ class ConfigCommandTest extends ZooKeeperTestHarness with Logging {
     assertEquals("[[1, 2], [3, 4]]", addedProps.getProperty("nested"))
   }
 
-  def doTestOptionEntityTypeNames(zkConfig: Boolean): Unit = {
-    val connectOpts = if (zkConfig)
-      ("--zookeeper", zkConnect)
-    else
-      ("--bootstrap-server", "localhost:9092")
-
+  def doTestOptionEntityTypeNames(): Unit = {
```

Comment:
This should be inlined into the test since it's only used in one place? Also, the removed test is no longer needed, I assume?

Reply:
Thanks for reminder! I added the removed test back, and only test "user" and "brokers" config type for `zkConfig`. Thanks.